### PR TITLE
fix dev support hotkeys for Ctrl+R , Shift+F10, Menu.

### PR DESCRIPTION
**Pull request by quincycs**
Currently the dev menu & refresh JS hotkeys don't fire always. This fixes that!

Test plan (required)

Held down buttons for 3 seconds to make sure app doesn't crash. Dev menu opens, and Ctrl+R refreshes app with new content.

### CODE REVIEW

**Comment by mpodwysocki** [comment]
LGTM 👍

File: ReactWindows/ReactNative/ReactPage.cs [hunk 1]
```diff
@@ -140,57 +144,33 @@ protected virtual ReactRootView CreateRootView()
...
-            {
-                if (e.Key == VirtualKey.Menu)
+                else if (e.VirtualKey == VirtualKey.Control)
                 {
-                    _reactInstanceManager.DevSupportManager.ShowDevOptionsDialog();
-                    e.Handled = true;
+                    _isControlKeyDown = e.EventType == CoreAcceleratorKeyEventType.KeyDown;
                 }
-                else if (e.Key == VirtualKey.Shift)
+                else if (_isShiftKeyDown && (e.VirtualKey == VirtualKey.F10 || e.VirtualKey==VirtualKey.Menu))
```
Comment (rozele) [inline ReactWindows/ReactNative/ReactPage.cs]: I think the intent is for menu to work without shift (that's the expected behavior on Android, at least).  I added Shift+F10 for keyboards without a Menu button.

**Comment by rozele** [comment]
Thanks @quincycs! Looks good, please do make the one change to ensure the same behavior for the Menu key though.

**Comment by quincycs** [comment]
@rozele thx,  honestly idk how to test menu button.  but good catch :)

**Comment by rozele** [comment]
:+1:

**Comment by rozele** [comment]
@quincycs I pulled this down and tried this out, and I was still noticing that the KeyUp events weren't firing.  It looks like the _isShiftKeyDown and _isCtrlKeyDown fields can get stuck in true as a result.

**Comment by quincycs** [comment]
image
oh man it was because of the Menu key!!  So if you press ctrl+R , virtualKey==Menu apparently.   

What's the purpose of Menu again?  How should a user push the "menu".  Is that like a fancy keyboard button that not all keyboards have?  I can try and find a keyboard with it if it is...ha, just not sure how to test it.

**Comment by quincycs** [comment]
@rozele with the above checkin I fixed what you were seeing.  But the menu key is kinda up in air.

**Comment by rozele** [comment]
Fair enough.  This is working much better than what we had before.

**Comment by rozele** [comment]
LGTM

**Pull request merged**